wiki/en/en-Command-Line-Options.md

```diff
@@ -31,7 +31,7 @@ You can see all possible options your version supports by starting Jamulus with
 |    `-c` |`--connect`        | Connect to given server address on startup, format `address[:port]` | (client only) |
 |    `-d` |`--discononquit`   | Disconnect all clients on quit | (server only) |
 |    `-e` |`--centralserver`  | Makes the server public and sets its genre. See also `-o`| (server only) [See list](Central-Servers). |
-|    `-f` |`--listfilter`     | Whitelist servers registering on the serverlist, format `ip address 1[;ip address 2]` | (central server only) [See note](Choosing-a-Server-Type#3-central). |
+|    `-f` |`--listfilter`     | Whitelist servers registering on the server list, format `ip address 1[;ip address 2]` | (central server only) [See note](Choosing-a-Server-Type#3-central). |
```

Comment:
As per the brief discussion on this, we should change this to say "Allow only listed servers to register on the server list..." I think.